python/delta_sharing/version.py

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 #
 
-__version__ = "0.3.0.dev0"
+__version__ = ""
```

Comment:
unnecessary change

Reply:
Sorry don't know how that got in there. Reverted.